README.md

```diff
@@ -3,12 +3,23 @@
 ## Developing
 
 To develop this application, you must have [NodeJS](https://nodejs.org) (20 or higher) and [npm](https://www.npmjs.com/) installed.
+You must also have access to a postgres database. If you have docker, you can spawn one by simply running:
+
+```sh
+npm run db:start
+```
```

Comment:
Can we add what connection string will need to be added to the `.env` file? Or even better, add it to `.env.example` and have an instruction to copy from `.env.example` to `.env`.

Reply:
Added